src/core/api/index.js

```diff
@@ -1,4 +1,5 @@
 /* global fetch */
+/* eslint-disable arrow-body-style */
```

Comment:
You're going to fight me for it! It's about to die https://github.com/mozilla/addons-frontend/issues/1252 I generally agree that single value returns should always use arrow body but long promise chains get really hard to read. In fact, when I first submitted the code for review, mark had trouble with it because it was hard to follow the `response` variable reference.